packages/relay/tsconfig.json

```diff
@@ -14,7 +14,8 @@
         "forceConsistentCasingInFileNames": true,
         "noImplicitAny": false,
         "declaration": true,
-        "strict": true
+        "strict": true,
+        "sourceMap": true
```

Comment:
I know the project is open source, but can we confirm that it's okay to build with this as a default?  As set production builds for mainnet would include `sourceMap` information.

Reply:
I recommend we add another tsconfig which is used for production builds:

**tsconfig.prod.json:**
```json
{
  "extends": "./tsconfig.json",
  "compilerOptions": {
    "sourceMap": false
  }
}
```

Reply:
But let's do that in a separate issue, the CI is really unstable right now and I believe we want to merge this PR before the next release?